src/components/sky-header/sky-header.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'sky-header',
+  templateUrl: 'sky-header.html',
+})
+export class SkyHeaderComponent implements OnInit {
```

Comment:
Where possible try to reuse code from:

https://github.com/samuelvisscher/skycoin/tree/develop/src/gui/static/src/app/components/layout/header

The main difference is the layout and the fact that the amount of coins and hours will be an input, instead of provided by a service.